talk/owt/sdk/include/cpp/owt/base/videorendererinterface.h

```diff
@@ -70,6 +74,30 @@ class VideoRenderWindow {
   Window wnd_;
 };
 #endif
+#endif
+#if defined(WEBRTC_LINUX)
+typedef void (*PFN_RETURN_BUFFER)(void *data, uint32_t bufid);
+/// libva surface
+typedef void* VADisplay;        /* window system dependent */
+typedef unsigned int VASurfaceID;
+struct VaSurface {
```

Comment:
Descriptions are required for all public interfaces.

Reply:
done.